lib/src/types.dart

```diff
@@ -23,9 +23,9 @@ class PostgrestException implements Exception {
   }) {
     return PostgrestException(
       message: (json['message'] ?? message) as String,
-      code: (json['code'] ?? '$code') as String,
+      code: '$code',
```

Comment:
I'm removing `json['code']`, because on select on a not existent table, the code in the body is `42p01`, which is not an `int`. So I'm changing to only use the REST status code. (https://bobcares.com/blog/postgresql-error-42p01/)

Reply:
I believe it's better we use the postgrest code, instead of the rest status code, so that it's better to know what's the issue